### PR TITLE
Use custom multigrid component

### PR DESCRIPTION
**Pull request by bthesorceror**
Multigrid from react-virtualized did not allow us to style the table graphs with our fancy scrollbars. Pulling that part of the library into our codebase allows for us to modify the scrollbar behaviour. This solves an issue with scrollbars in firefox.

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by ebb-tide** [comment]
Very cool, thanks for doing this!

File: ui/src/shared/components/MultiGrid/MultiGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,810 @@
...
+            scrollToColumn,
+            scrollToRow,
+            scrollTop,
+          })}
+        </div>
+      </div>
+    )
+  }
+
+  public cellRendererBottomLeftGrid = ({rowIndex, ...rest}) => {
```
Comment (jaredscheib) [inline ui/src/shared/components/MultiGrid/MultiGrid.tsx]: what do you think about adding input and output types to all of these non-React methods?

File: ui/src/shared/components/FancyScrollbar.tsx [hunk 2]
```diff
@@ -1,40 +1,76 @@
...
+    this.ref = React.createRef<Scrollbars>()
   }
 
-  static defaultProps = {
-    autoHide: true,
-    autoHeight: false,
-    setScrollTop: () => {},
+  public updateScroll() {
+    const ref = this.ref.current
+    if (ref && _.isNumber(this.props.scrollTop)) {
```
[the remark this answers is not in the document]
Comment (bthesorceror) [reply ui/src/shared/components/FancyScrollbar.tsx]: this simply is a check to see if the value is not null, might just switch this for `!_.isNil(...)`

File: ui/src/shared/components/MultiGrid/CellMeasurerCacheDecorator.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  cellMeasurerCache: CellMeasurerCache
+  columnIndexOffset: number
+  rowIndexOffset: number
+}
+
+interface IndexParam {
+  index: number
+}
+
+export default class CellMeasurerCacheDecorator {
```
[the remark this answers is not in the document]
Comment (bthesorceror) [reply ui/src/shared/components/MultiGrid/CellMeasurerCacheDecorator.ts]: 👍

File: ui/src/shared/components/MultiGrid/MultiGrid.tsx (hunk 1 above)
Comment (bthesorceror) [reply ui/src/shared/components/MultiGrid/MultiGrid.tsx]: 👍

**Pull request merged**